css/index.css

```diff
@@ -143,4 +143,42 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+/* my start */
+html {width: 800px; margin: auto;}
+header img {display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 800px; margin-top: 20px 2%;
+}
+header {margin-bottom: 2%;}
+.navBar {width: 800px;
+    display: flex;
+    align-items: baseline;
+    padding-top: 20px;
+    margin-bottom: 20px;
+    font-size: 14px; margin-left: auto;
+    margin-right: auto;
+}
+.navBar a {justify-content: space-between;text-decoration: none; color: black; width: 20%;}
+.navBar img {width: 20%; margin-right: 5%;
+}
```

Comment:
Nice! You used selectors instead of adding an additional class or id on the boxes of the index page.

Reply:
Classes and ids feel like they clutter up the html. Its fun to try and not use them unless they're needed.